sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -696,7 +701,7 @@ class Analyzer(
               s"avoid errors. Increase the value of ${SQLConf.MAX_NESTED_VIEW_DEPTH.key} to work " +
               "around this.")
           }
-          executeSameContext(child)
+          executeSameContext(child, None)
```

Comment:
No great reason. I just used None for everything, except the top level, because it is very difficult to wire the tracker here without refactoring a lot of code.